pages/docs/tutorials/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Create an AsyncAPI Document for a Slackbot with WebSockets
+description: In this tutorial, you'll learn how to create an AsyncAPI document designed for a Slack application that operates in Socket Mode using the WebSockets protocol.
+weight: 140
+---
+
+## Introduction
+In this tutorial, you will learn how to generate an AsyncAPI document designed for a Slack application that operates in Socket Mode. The aim is to help you grasp a real-world application of AsyncAPI with the WebSocket protocol.
```

Comment:
```suggestion
In this tutorial, you will learn how to write an AsyncAPI document designed for a Slack application that operates in Socket Mode. The aim is to help you grasp a real-world application of AsyncAPI with the WebSocket protocol. You will learn how to write AsyncAPI document for a consumer-only application receiving a stream of messages from a Websocket server. You will also learn why AsyncAPI bindings feature exist and how to use it.
```

I think it is good to explain also what someone will learn, in raw (no matter if slack or some other service is used as example)

> You will also learn why AsyncAPI bindings feature exist and how to use it.

have to be text without link but in future we can link to whatever comes out from https://github.com/asyncapi/website/pull/2042